build/_k-restore.shade

```diff
@@ -6,5 +6,5 @@ k-restore
 
 default restore_options='${E("NUGET3_restore_options")}'
 
-exec program='cmd' commandline='/C dnu restore${restore_options}' if='!IsMono'
+exec program='cmd' commandline='/C dnu restore --parallel ${restore_options}' if='!IsMono'
```

Comment:
Ehh, makes this easier to read. Plus it requires specifying the env variable with a leading space which seems unusual.